enclave-runtime/src/tls_ra/mod.rs

```diff
@@ -44,11 +46,12 @@ impl TcpHeader {
 }
 
 /// Indicates the payload content type.
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Decode, Encode, MaxEncodedLen)]
 pub enum Opcode {
-	ShieldingKey = 0,
-	StateKey = 1,
-	State = 2,
+	ShieldingKey,
+	StateKey,
+	State,
+	LightClient,
```

Comment:
Clippy gave me warnings here, and as `From<u8>` is implemented manually too, I don't think it adds a lot of value. This part should be improved anyhow.